test.py

```diff
@@ -379,7 +379,7 @@ async def create_test(self, shortname: str, casename: str, suite, args) -> None:
                 )
                 _, stderr = await asyncio.wait_for(process.communicate(), options.timeout)
 
-                case_list = [case[:-1] for case in stderr.decode().splitlines() if case.endswith('*')]
+                case_list = [case[:-1] for case in stderr.decode().splitlines() if case.endswith('*') and not case.strip().startswith('_')]
```

Comment:
yes, this is the magic here, could you please accompany this statement with some comment explaining why we make the test case with name starting with `_` a special case. better off adding an example.

Reply:
Added a comment with explanation.